com.unity.shadergraph/Editor/Data/Nodes/Input/Lighting/GetMainLightNode.cs

```diff
@@ -0,0 +1,40 @@
+using System.Reflection;
+using UnityEngine;
+
+namespace UnityEditor.ShaderGraph
+{
+    [Title("Input", "Lighting", "Main Light")]
+    class GetMainLightNode : CodeFunctionNode
+    {
+        public GetMainLightNode()
+        {
+            name = "Main Light";
```

Comment:
I recommend adding a line like the following at line 12:
`synonyms = new string[] { "sun", "direction" };`
These synonyms are used by the Create Node menu's search filter and can help users find your node even if they don't know the exact name.  Use whatever synonyms you think people might type in the search field when looking for your node.

Reply:
Oh that's nice ! i added the synonym